docs/ReleaseNotes.txt

```diff
@@ -1,3 +1,74 @@
+=== v1.22.0 === (15 November 2023)
+
+ISPC release with template operators support; improved debugging experience of ISPC code on Windows; multiple
+stability and performance fixes and more. The release is based on patched LLVM 16.0.6.
+
+ISPC distribution changes:
+
+ - ISPC binaries were compiled with LTO by Clang/LLVM toolchain on all supported platforms and architectures using
+   superbuild. ISPC binaries got faster a few percent in average.
+ - Examples were excluded from ISPC archives. They are placed alongside as separate archives ispc-examples-v1.22.0.zip
+   and ispc-examples-v1.22.0.tar.gz.
+
+Language changes:
+
+ - Added support for template operators.
+ - Revised the usage of function specifiers with templates. For more details please refer to Function Templates
+   section of documentation.
+
+Infrastructure changes:
+
+ - Release built with LTO (expect aarch64 Linux).
+ - Supported building ISPC with LLVM 17.
```

Comment:
Would be good to give a reason why the release is not with LLVM 17 - i.e. GPU support is incomplete, there are known regressions, was not tested throughly enough, etc.